src/communities/communities.sql.js

```diff
@@ -50,46 +49,44 @@ export const getCommunities = `
         BOOK b ON c.book_id = b.book_id  -- 커뮤니티의 book_id를 사용하여 책 정보 조회
     WHERE 
         c.is_deleted = 0
-    LIMIT ?, ?;`;
+    LIMIT ? OFFSET ?;`;
 
-// 나의 참여 모임 리스트 조회 쿼리 (최신 메시지 온 순으로 정렬)
-export const getMyCommunities =  `
-    SELECT cu.*, c.*, b.title, b.link
-    FROM COMMUNITY_USERS cu
-    LEFT JOIN MESSAGE m ON cu.community_id = m.community_id
-    LEFT JOIN COMMUNITY c ON cu.community_id = c.community_id  -- 커뮤니티 테이블 조인
-    LEFT JOIN BOOK b ON c.book_id = b.book_id
-    WHERE cu.user_id = ? AND cu.is_deleted = 0
-    GROUP BY cu.community_id
-    ORDER BY 
-        MAX(m.created_at) DESC,
-        COUNT(m.message_id) ASC  -- 메시지가 없는 경우 아래로 정렬
-    LIMIT ? OFFSET ?;
-`;
+// 나의 참여 모임 리스트 조회 쿼리 (최신 메시지 온 순으로 정렬) + 안읽음 개수 카운트
+export const getMyCommunities = `
+SELECT
+    c.community_id, c.capacity, c.tag, c.location,
+    COALESCE(currentCount.cnt, 0) AS currentCount,
+    COALESCE(unread.cnt, 0) AS unreadCnt,
+    b.title, b.image_url as bookImg,
+    recent_msg.latest_message_time
+FROM COMMUNITY c
+LEFT JOIN (
+    SELECT community_id, COUNT(*) AS cnt
+    FROM COMMUNITY_USERS
+    GROUP BY community_id
```

Comment:
```suggestion
    SELECT community_id, COUNT(*) AS cnt
    FROM COMMUNITY_USERS
    WHERE COMMUNITY_USERS.is_deleted = false
    GROUP BY community_id
```

COMMUNITY_USER is_deleted 반영한 쿼리로 수정 부탁드려요!